sdcm/cluster_k8s/__init__.py

```diff
@@ -412,8 +427,26 @@ def deploy_scylla_manager(self, pool_name: str = None) -> None:
                 pool_name = self.AUXILIARY_POOL_NAME
             LOGGER.info("Deploy scylla-manager")
 
-            helm_affinity = get_helm_pool_affinity_values(self.POOL_LABEL_NAME, pool_name) if pool_name else {}
+            helm_affinity = get_helm_pool_affinity_values(self.POOL_LABEL_NAME, pool_name)
```

Comment:
Could you revert it back for minikube and kind case

Reply:
Done